### ISSUE TITLE
Code Oops Reminder

### ISSUE THREAD

**Issue by sydneywalcoff** [hackforla/expunge-assist#1142]

**Comment by eruan8888** [comment]
Design can be found [here](https://www.figma.com/design/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=16837-61178&t=CwNcrTxclQyTyvz2-1)

**Comment by sydneywalcoff** [comment]
Current Figma content isn't fully accurate so Content is re-evaluating

**Comment by sydneywalcoff** [comment]
@raeraeicky yes! I just have some last minute touches. It should be live today or tomorrow.
